{{cookiecutter.project_slug}}/Makefile

```diff
@@ -82,6 +77,10 @@ LOCAL_PORT?=2211
 # run Jupyter Notebook), `lab` (to run JupyterLab):
 #   make jupyter JUPYTER_MODE=LAB
 JUPYTER_MODE?=notebook
+# Maximum running time for Jupyter. Available opitons:
+# `1d`, `1h`, `8h`, `0` (to disable):
+#   make jupyter JUPYTER_LIFE_SPAN=4h
+JUPYTER_LIFE_SPAN?=0
```

Comment:
I noticed that some of my test jupyters don't get killed by the cleanup mechanism :flushed: So we restrict their lifetime to `1h` in tests.